new_platforms/include/oeenclave.h

```diff
@@ -2,39 +2,41 @@
 /* Licensed under the MIT License. */
 #pragma once
 #ifndef _OE_ENCLAVE_H
-# error include <openenclave/enclave.h> instead of including oeenclave.h directly
+#error include <openenclave/enclave.h> instead of including oeenclave.h directly
```

Comment:
why the whitespace change?  This makes it harder to read in my opinion.   Is this the result of running a tool?  (if so, ok)

Reply:
Reverted.

OE have a clang auto-formatting rules file. This may have been a result of that tool somehow getting invoked (probably via VS, which has support for this). We should run a pass of the formatter through our code in a separate whitespace-only commit.